scripts/gv_control_loop_gym.py

```diff
@@ -11,19 +11,23 @@
 from gym_gridverse.representations.observation_representations import (
     DefaultObservationRepresentation,
 )
+from gym_gridverse.representations.state_representations import (
+    DefaultStateRepresentation,
+)
 
 
 def make_env(id_or_path: str) -> GymEnvironment:
     try:
         print('Loading using gym.make')
         env = gym.make(id_or_path)
 
-    except gym.error.UnregisteredEnv:
+    except gym.error.Error:
```

Comment:
Any reason why this error became more generic?  Did you encounter some errors which were not covered by UnregisteredEnv?

Reply:
As I mentioned in the original pull request, the issue is that gym won't throw an UnregisteredEnv Error if the environment id is "malformed", it'll just throw a standard Error. In realistic use cases where you would pass some yaml filepath to the function, the filepath won't be a valid gym id (valid being of the form ^(?:[\w:-]+\/)?([\w:.-]+)-v(\d+)$), so gym won't even check if such an environment is registered

Reply:
Oh, I see, sorry, I missed that part.  It's annoying that we don't have a good system in place to test scripts;  I guess that's why that issue was not caught before.